addOns/reports/src/main/java/org/zaproxy/addon/reports/automation/ReportJob.java

```diff
@@ -129,6 +129,22 @@ public void verifyParameters(AutomationProgress progress) {
             }
             this.getData().setSections(list);
         }
+
+        list = getJobDataList(jobData, "sites", progress);
+        if (!list.isEmpty()) {
+            List<String> validSites = ExtensionReports.getSites();
+            for (String site : list) {
+                if (!validSites.contains(site)) {
```

Comment:
This is being done too early, in most cases the plan is going to be run with an empty session so no sites yet. I think this validation should be done only when running the job (as it already is).